public/app/features/scenes/variables/types.ts

```diff
@@ -13,16 +13,9 @@ export interface SceneVariableState extends SceneObjectStatePlain {
   loading?: boolean;
   error?: any | null;
   description?: string | null;
-  //text: string | string[];
-  //value: string | string[]; // old current.value
 }
 
 export interface SceneVariable<TState extends SceneVariableState = SceneVariableState> extends SceneObject<TState> {
-  /**
-   * Should return a string array of other variables this variable is using in it's definition.
-   */
-  getDependencies?(): string[];
-
```

Comment:
Can now remove this as variables use exact same abstraction as other scene objects to declare their dependence

Reply:
Nice!